Functions.Templates/Bindings/bindings.json

```diff
@@ -1747,6 +1747,639 @@
                     "help": "$bot_out_secret_help"
                 }
             ]
-        }
+        },
+        {
+            "type": "token",
+            "displayName": "$token_displayName",
+            "direction": "in",
+            "enabledInTryMode": false,
+            "documentation": "$content=Documentation\\Token.md",
+            "filters": ["MSGraph"],
+            "settings": [
+                {
+                    "name": "name",
+                    "value": "string",
+                    "defaultValue": "token",
+                    "required": true,
+                    "label": "$token_in_name_label",
+                    "help": "$token_in_name_help"
+                },
+                {
+                    "name": "Resource",
+                    "value": "string",
+                    "required": false,
+                    "label": "$token_in_resource_label",
+                    "help": "$token_in_resource_help"
+                },
+                {
+                    "name": "PrincipalId",
+                    "value": "appSetting",
```

Comment:
I have a separate component that handles these, because the process for retrieving a new principal ID app setting is very different from the existing "new" button beside the app settings. Should I still merge the two?